tests/helper_functions.py

```diff
@@ -87,6 +86,31 @@ def score_model_in_sagemaker_docker_container(
     return _evaluate_scoring_proc(proc, 5000, data, content_type, activity_polling_timeout_seconds)
 
 
+def pyfunc_generate_dockerfile(model_uri=None, extra_args=None):
```

Comment:
```suggestion
def pyfunc_generate_dockerfile(output_directory, model_uri=None, extra_args=None):
```

Can we add `output_directory` and pass a temporary directory created by pytest to this argument?

Reply:
Yep, it will be more convenient to do so
Done